package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/polaris-tokens",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
You can revert this change, as the version number will be bumped by `yarn version` (which we perform on master directly, and may require an admin to do it).